Networking/NetworkingTests/Responses/blaze-campaigns-list-success.json

```diff
@@ -3,7 +3,7 @@
     {
       "id": "34518",
       "status": "rejected",
-      "start_time": "2024-01-05T00:00:00.000Z",
+      "start_time": "2024-09-02T04:58:03.000Z",
```

Comment:
It's not fully clear to me why this change 🤔

Reply:
I just updated the value to unit test the date [here](https://github.com/woocommerce/woocommerce-ios/pull/13807/files#diff-3240e7d366cbcc768e8ea2c53db05cc5f20c2a572e496c646f1f3d9e8743c548R33).